src/meta/src/stream/stream_graph.rs

```diff
@@ -918,7 +909,7 @@ impl ActorGraphBuilder {
                             FragmentDistributionType::Hash
                         } as i32,
                         actors,
-                        vnode_mapping: None,
+                        ..Default::default()
```

Comment:
Better not to use `Default:default()` here, or we may forget to update this code if we update the fields in the future. We may explicitly use default value for the remaining fields, and leave a comment about at which stage it'll be filled.